core/templates/css/oppia.css

```diff
@@ -525,6 +525,10 @@ p:last-child {
   vertical-align: middle;
 }
 
+.oppia-welcome-guide-browser-image span {
```

Comment:
Move this to the file where its used inside <style> tag

Reply:
fixed